src/Kalkulator/Kalkulator.tsx

```diff
@@ -1,62 +1,185 @@
+            return 0;
+        }
+    };
+    const harEndretTapteDagsverk = nåværendeTapteDagsverk !== undefined;
+    const harEndretSykefraværsprosent = nåværendeSykefraværsprosent !== undefined;
 
-    const harEndretTapteDagsverk = tapteDagsverk !== undefined;
+    const labelsNåværendeTapteDagsverkEllerProsent =
+        antallTapteDagsverkEllerProsent === AntallTapteDagsverkEllerProsent.SYKEFRAVÆRSPROSENT
+            ? 'Nåværende sykefravær (%)'
+            : 'Nåværende antall tapte dagsverk siste 12 mnd';
+    const labelsØnsketTapteDagsverkEllerProsent =
+        antallTapteDagsverkEllerProsent === AntallTapteDagsverkEllerProsent.SYKEFRAVÆRSPROSENT
+            ? 'Ønsket sykefravær (%)'
+            : 'Ønsket antall tapte dagsverk siste 12 mnd';
 
+    const setVerdiAntallTapteDagsverkEllerProsent = (verdi: number) => {
+        if (
+            antallTapteDagsverkEllerProsent === AntallTapteDagsverkEllerProsent.SYKEFRAVÆRSPROSENT
+        ) {
+            setNåværendeSykefraværsprosent(verdi);
+        } else {
+            setNåværendeTapteDagsverk(verdi);
+        }
+        //console.log(getSykefraværsprosentSiste4Kvartaler(restSykefraværshistorikk.data));
+    };
+    const setØnsketVerdiAntallTapteDagsverkEllerProsent = (verdi: number) => {
+        if (
+            antallTapteDagsverkEllerProsent === AntallTapteDagsverkEllerProsent.SYKEFRAVÆRSPROSENT
+        ) {
+            setØnsketSykefraværsprosent(verdi);
+        } else {
+            setØnsketTapteDagsverk(verdi);
+        }
+        //console.log(getSykefraværsprosentSiste4Kvartaler(restSykefraværshistorikk.data));
+    };
     useEffect(() => {
         if (restSykefraværshistorikk.status === RestStatus.IkkeLastet) {
-            setTapteDagsverk(undefined);
+            setNåværendeTapteDagsverk(undefined);
+            setØnsketTapteDagsverk(undefined);
+            setMuligeDagsverk(undefined);
+            setNåværendeSykefraværsprosent(undefined);
         }
     }, [restSykefraværshistorikk]);
 
     useEffect(() => {
-        if (restSykefraværshistorikk.status === RestStatus.Suksess && !harEndretTapteDagsverk) {
-            const tapteDagsverkSiste4Kvartaler = getAntallTapteDagsverkSiste4Kvartaler(
-                restSykefraværshistorikk.data
-            );
-            if (tapteDagsverkSiste4Kvartaler === 'erMaskertEllerHarIkkeNokData') {
-                setTapteDagsverk(undefined);
-                setSkalViseDefaultTapteDagsverk(false);
-            } else {
-                setTapteDagsverk(tapteDagsverkSiste4Kvartaler);
-                setSkalViseDefaultTapteDagsverk(true);
+        console.log('harendrettapte ' + harEndretTapteDagsverk);
+        console.log('harendretsyfoprosent ' + harEndretSykefraværsprosent);
+        if (
+            restSykefraværshistorikk.status === RestStatus.Suksess &&
+            (!harEndretTapteDagsverk || !harEndretSykefraværsprosent)
+        ) {
+            switch (antallTapteDagsverkEllerProsent) {
```

Comment:
Her har vi en `switch`/`case` med bare to tilfeller, lurer på om det er mer logisk med `if` og `else`

Reply:
skal gjøre den til if else